kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Scheduler.scala

```diff
@@ -258,34 +252,37 @@ object Scheduler {
         coreWorkers: Int,
         minWorkers: Int,
         maxWorkers: Int,
-        scheduleTries: Int,
         scheduleStride: Int,
+        stealStride: Int,
         virtualizeWorkers: Boolean,
         timeSliceMs: Int,
+        cycleNs: Int,
         enableTopJMX: Boolean,
         enableTopConsoleMs: Int
     )
     object Config {
         val default: Config = {
             val cores             = Runtime.getRuntime().availableProcessors()
-            val coreWorkers       = Math.max(1, Flag("coreWorkers", cores))
+            val coreWorkers       = Math.max(1, Flag("coreWorkers", cores * 10))
```

Comment:
Start with more workers to avoid having to wait for the concurrency regulator to adapt.